src/main/java/com/github/maxopoly/kira/api/APISession.java

```diff
@@ -153,6 +153,14 @@ public void sendReplacementToken(APIToken token) {
 		json.put("expires", token.getExpirationTime());
 		connection.send(json.toString());
 	}
+	
+	public void sendInGameCommandResponse(String command, String response) {
+		JSONObject json = new JSONObject();
+		json.put("type", "ingame-response");
+		json.put("command", command);
```

Comment:
The command used is not a unique identifier imo. The same command may lead to a different output when run repeatedly. Instead the API user should be providing some kind of token unique to the particular command being run, which is then used in the reply. A simple implementation for clients for this would for example be the usage of an incrementing counter